apps/src/code-studio/pd/professional_learning_landing/LandingPage.jsx

```diff
@@ -144,8 +156,32 @@ function LandingPage({
     dispatch(asyncLoadSectionData());
     dispatch(asyncLoadCoteacherInvite());
 
+    const fetchParticipantData = async () => {
+      setLoadingWorkshopsAsParticipant(true);
+      try {
+        const response = await fetch('/api/v1/pd/workshops_user_enrolled_in', {
+          method: 'GET',
+          headers: {
+            'Content-Type': 'application/json',
+            'X-CSRF-Token': await getAuthenticityToken(),
+          },
+        });
+
+        if (response.ok) {
+          const jsonData = await response.json();
+          console.log('jsonData', jsonData);
```

Comment:
Can the console log statement here be deleted?

Reply:
🤦‍♀️